lighthouse-core/gather/gather-runner.js

```diff
@@ -119,7 +120,10 @@ class GatherRunner {
     await driver.registerPerformanceObserver();
     await driver.dismissJavaScriptDialogs();
     await driver.registerRequestIdleCallbackWrap(options.settings);
-    if (resetStorage) await driver.clearDataForOrigin(options.requestedUrl);
+    if (resetStorage) {
+      await driver.getImportantStorageWarning(options.requestedUrl, LighthouseRunWarnings);
```

Comment:
would prefer this to actually return a string, or null, and push to warnings array if warning exists